src/buttons.ts

```diff
@@ -50,18 +50,10 @@ function replaceVoteButton (program: Program): void {
 						headers: { "X-KA-FKey": getCSRF() },
 						credentials: "same-origin"
 					}).then((response: Response): void => {
-						if (response.status !== 204) {
-							response.json().then((res: any): void => {
-								if (res.error) {
-									alert("Failed with error:\n\n" + res.error);
-								}
-							}).catch(() => {
-								alert(`Voting failed with status ${response.status}`);
-							});
+							if (response.status === 204) { return; }
```

Comment:
@lukekrikorian This is the change I'm talking about my comment on the PR.

Reply:
I didn't realize that was the use-case, apologies! I had assumed it was simply something left over from debugging the code. I'm willing to rework this unless you wanted to, I don't think it will be too difficult.